### PR TITLE
Feature/eslint

### PR DESCRIPTION
**Pull request by 0candy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "description": "Generate loopback api definitions",
   "main": "lib/get-api-def.js",
   "scripts": {
-    "pretest": "jscs . && jshint .",
-    "test": "mocha"
```
Comment (superkhau) [inline package.json]: "lint": "eslint .",

File: package.json [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
+    "test": "mocha",
+    "posttest": "eslint ."
```
Comment (superkhau) [inline package.json]: `"posttest": "npm run lint"`

**Comment by superkhau** [comment]
One minor fix, LGTM, no more review needed.

File: package.json (hunk 2 above)
Comment (bajtos) [inline package.json]: Yes please.

```json
{
  "scripts": {
    "lint": "eslint .",
    "test": "mocha",
    "posttest": "npm run lint"
  }
}
```

**Comment by bajtos** [comment]
LGTM, :shipit:

**Pull request merged**